src/dbwrapper.h

```diff
@@ -132,10 +132,6 @@ class CDBIterator
         return true;
     }
 
-    unsigned int GetKeySize() {
```

Comment:
@paveljanik Good point! Reference in `src/coins.h` removed. Squashed and pushed as well :-)